includes/cognitive-services-bing-entity-search-signup-requirements.md

```diff
@@ -10,10 +10,10 @@ ms.date: 05/08/2020
 
 Start using the Bing Entity Search API by creating one of the following Azure resources.
 
-### [Bing Entity Search resource](https://ms.portal.azure.com/#create/Microsoft.CognitiveServicesBingEntitySearch)
+### [Bing Entity Search resource](https://ms.portal.azure.com/#create/microsoft.bingsearch)
```

Comment:
I'm not actually sure if we are allowed to change this file because I don't think it is owned by our team... @HeidiSteen thoughts? How does review usually work if we find a bug in a partner team's docs like this?

Reply:
@jonathanserbent Since you removed the use of this includes file in our example., I would undo the changes we did here. As Aaron mentioned they are trying to keep these docs pointing towards the old deprecated instructions for now so we shouldn't mess with it.

Reply:
@Careyjmac done and done :)